### ISSUE TITLE
Calculating SAC on metagenome clusters

### ISSUE THREAD

**Issue by nmb85** [dib-lab/wort#36]
@luizirber, one more thing for today (not intending to distract you), it would be really interesting if you could calculate the species accumulation curve (SCA) for hash sets in clusters of metagenomes in your monster wort database. For example, when looking at soil metagenomes as a cluster, you could build a matrix of hashes (such as [here](https://github.com/dib-lab/sourmash/issues/1189)), calculate different orders of intersection between hash sets from the soil metagenomes, and then plot an SCA from the hashes. While this might be impossible with kmers, and species tallies are corrupted by incomplete annotation due to incomplete databases, hashes might give you a chance to get an accurate SCA based on plotting the effect of incrementally adding hash sets and seeing the change in intersection sets. See equation 3 in [this paper](https://www.jstor.org/stable/10.1086/678125?seq=1#metadata_info_tab_contents) for a definitive explanation. Then you could efficiently use _all the data in the SRA and JGI dbs_ to estimate if the species count based on current soil metagenome is "open" or "closed", that is, whether or not we've collected enough data to capture all species in soil metagenomes (or some other interesting biome). Although I'm not a soil biologist, I think that's a major question in their field. Other biomes might be interesting too. Not sure if anyone has tried this with raw kmers, but it would seem too gargantuan of a task. Hashes might make this problem tractable?

**Comment by luizirber** [comment]
That is a really good idea... and a monstrous matrix :rofl: 

I'll work on sharing all the sigs in a couple of weeks, but it is not something I can tackle at the moment :cry:

**Comment by ctb** [comment]
yes! we explored this quite a bit a while back for tara, see https://github.com/ctb/2017-sourmash-rarefy/blob/master/tara-rarefy.ipynb for an example. Haven't looked at the code in a while tho ;).

**Comment by nmb85** [comment]
Have you already seen this?
https://ieeexplore.ieee.org/abstract/document/9139876
